pkg/azure/plugin_integration_test.go

```diff
@@ -90,44 +91,46 @@ func TestBasicPermitListOps(t *testing.T) {
 			Protocol:  6,
 		},
 	}
-	addPermitListResp, err := s.AddPermitListRules(ctx, &paragliderpb.AddPermitListRulesRequest{Rules: rules, Namespace: "default", Resource: vmID})
+	addPermitListResp, err := s.AddPermitListRules(ctx, &paragliderpb.AddPermitListRulesRequest{Rules: rules, Namespace: namespace, Resource: vmID})
 	require.NoError(t, err)
 	require.NotNil(t, addPermitListResp)
 
 	// Assert the NSG created is equivalent to the pl rules by using the get permit list api
-	getPermitListResp, err := s.GetPermitList(ctx, &paragliderpb.GetPermitListRequest{Resource: vmID, Namespace: "default"})
+	getPermitListResp, err := s.GetPermitList(ctx, &paragliderpb.GetPermitListRequest{Resource: vmID, Namespace: namespace})
 	require.NoError(t, err)
 	require.NotNil(t, getPermitListResp)
 
 	assert.ElementsMatch(t, getPermitListResp.Rules, rules)
 
 	// Delete permit list rule
-	deletePermitListResp, err := s.DeletePermitListRules(ctx, &paragliderpb.DeletePermitListRulesRequest{RuleNames: []string{rules[0].Name}, Namespace: "default", Resource: vmID})
+	deletePermitListResp, err := s.DeletePermitListRules(ctx, &paragliderpb.DeletePermitListRulesRequest{RuleNames: []string{rules[0].Name}, Namespace: namespace, Resource: vmID})
 	require.NoError(t, err)
 	require.NotNil(t, deletePermitListResp)
 
 	// Assert the rule is deleted by using the get permit list api
-	getPermitListResp, err = s.GetPermitList(ctx, &paragliderpb.GetPermitListRequest{Resource: vmID, Namespace: "default"})
+	getPermitListResp, err = s.GetPermitList(ctx, &paragliderpb.GetPermitListRequest{Resource: vmID, Namespace: namespace})
```

Comment:
Curious if changes like this are also a bug fix.

Reply:
Nope I just made it into a variable because I made a function that takes in the namespace as an arg.